magpie/owsrequest.py

```diff
@@ -8,8 +8,7 @@
 from magpie.api.requests import get_multiformat_any
 from magpie.utils import get_logger, CONTENT_TYPE_JSON, CONTENT_TYPE_PLAIN, CONTENT_TYPE_FORM, is_json_body, get_header
 from typing import TYPE_CHECKING
-# noinspection PyUnresolvedReferences
-import lxml.etree
+import lxml.etree  # nosec: B410 # module safe but bandit flags it : https://github.com/tiran/defusedxml/issues/38
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'lxml.etree'](https://app.codacy.com/app/fmigneault/Magpie/pullRequest?prid=4650806)